### PR TITLE
Add OWNERS file

### PR DESCRIPTION
**Pull request by valen-mascarenhas14**
#2 added

### CODE REVIEW

**Comment by valen-mascarenhas14** [comment]
@mkumatag  Could you add me (**valen-mascarenhas14**) & Amit (**ghatwala**) as the members of the ppc64le-cloud org ?

**Review by ghatwala — APPROVED** [review]

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+reviewers:
+  - ghatwala
+  - valen-mascarenhas14
+approvers:
+  - ghatwala
+  - valen-mascarenhas14
```
Comment (mkumatag) [inline OWNERS:6]: add the new line in the end

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: OWNERS (hunk 1 above)
Comment (valen-mascarenhas14) [reply OWNERS]: added @mkumatag

**Comment by mkumatag** [comment]
> ghatwala

invitation sent for both of you folks, please accept it.

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (mkumatag) [inline OWNERS:7]: I still see this 
image
symbol!

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: OWNERS (hunk 2 above)
Comment (valen-mascarenhas14) [reply OWNERS]: fixed it @mkumatag

**Review by mkumatag — APPROVED** [review]
/lgtm

please accept the invites and fix the failing condition to merge the PR

**Pull request merged**